plaidml2/edsl/edsl_test.cc

```diff
@@ -756,6 +755,9 @@ module {
 }
 
 TEST(CppEdsl, GlobalMin) {
+  if (vertexai::env::Get("PLAIDML_EE") == "1") {
+    FAIL() << "Assertion failed: (map.getNumInputs() == mapOperands.size() && \"inconsistent index info\")";
```

Comment:
Huh?

Reply:
There area a few tests that cause this assertion to fail. Let's chat about it since it relates to how we translate AffinePolynomials into AffineExprs